### PR TITLE
Batch Installation EULA Fix

### PR DESCRIPTION
**Pull request by KSchmeeds**
Fix accepting the EULA for batch installations.

### CODE REVIEW

File: Oracle/VirtualBoxExtPack.munki.recipe [hunk 1]
```diff
@@ -120,7 +120,7 @@ then
 	exit 1
 fi
 
-echo y | $VBM extpack install --replace \
+	$VBM extpack install --accept-license=56be48f923303c8cababb0bb4c478284b688ed23f16d775d729b89a2e8e5f9eb --replace \
```
Comment (jessepeterson) [inline Oracle/VirtualBoxExtPack.munki.recipe]: Kill the leading space here, then should be good.

**Comment by jessepeterson** [comment]
Have you tested to see that this works in a full autopkg run-deploy-install cycle? I don't have time right now :(

**Comment by jessepeterson** [comment]
Also curious — was there a new update to VirtualBox? Which version broke this?

**Comment by KSchmeeds** [comment]
I can't remember exactly which version it was, but I think it was when 5.2 got released that I started seeing this issue.
I tested this fix locally on my machine and I no longer got the warning about accepting the license agreement.

**Comment by KSchmeeds** [comment]
Hi Jesse,
What are your thoughts on accepting this pull request?

**Comment by jessepeterson** [comment]
Apologies on the delay. Thanks @KSchmeeds!

**Pull request merged**